### PR TITLE
Make it work

### LINKED ISSUE #3: `git tag` and GitHub Actions research
Findings about `git tag -l --points-at HEAD`

## When pushing a tag

- The command returns the **pushed tag**
- even if it's not the latest available commit or tag

**but**

- after `fetch --tags`, it may change to a more recent tag **on the same commit**

## On schedule or branch push

- The command returns nothing

**unless**

- the user used `action/checkout@v2` with `fetch-depth=0`
- or we/they called `fetch --tags`

# Therefore

As long as the user doesn't use `fetch-depth=0`

- The command can be used to detect the pushed tag, if any.
- Otherwise, it can be used to detect the latest tag on the `schedule` or `branch` **after** `fetch --tags`

### PR DESCRIPTION
**Pull request by fregante**
Closes https://github.com/fregante/daily-version-action/issues/3

We got it! 🎉 

I ran these tests on `schedule`

## HEAD already tagged

The linebreak has been fixed by https://github.com/fregante/daily-version-action/pull/4/commits/cbaf2353c8d96b081cfc29ab9b558d4d4bd32c79

### CODE REVIEW

**Comment by fregante** [comment]
@notlmn this appears to work now and it includes minimal tests.

I'll leave this open in case you want to take a look before I merge

File: index.js [hunk 1]
```diff
@@ -1,20 +1,39 @@
...
+	if (process.env.GITHUB_REF.startsWith('refs/tags/')) {
+		const pushedTag = process.env.GITHUB_REF.replace('refs/tags/', '');
+		console.log('::set-output name=version::' + process.env.GITHUB_REF.replace('refs/tags/', ''));
+		console.log('Run triggered by tag `' + pushedTag + '`. No new tags will be created by `daily-version-action`.');
 		return;
 	}
 
-	const {stdout: version} = await execFile('npx', ['daily-version']);
+	// Look for tags on the current commit
+	await execFile('git', ['fetch', '--depth=1', 'origin', 'refs/tags/*:refs/tags/*']);
```
Comment (notlmn) [inline index.js:15]: Does `execFile` need `encoding` parameter similar to https://github.com/fregante/daily-version/commit/fa34122396040218fc2b31edbb2cb1888a1fce29#diff-168726dbe96b3ce427e7fedce31bb0bcR9-R11?

**Review by notlmn — APPROVED** [review]
Apart from the review, LGTM.

File: index.js (hunk 1 above)
Comment (fregante) [reply index.js:15]: For whatever reason, [`execFile`](https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback) defaults to `string` while [`execFileSync`](https://nodejs.org/api/child_process.html#child_process_child_process_execfilesync_file_args_options) uses `Buffer` 🤷‍♂️

**Comment by fregante** [comment]
Thank you for reviewing!

**Pull request merged**